pyproject.toml

```diff
@@ -36,7 +36,7 @@ boto3 = "^1.26.125"
 botocore="^1.35.54"
 gunicorn = "^20.1.0"
 sqlalchemy = "2.0.35"
-instructor = "1.6.3"
+instructor = "^1.5.2"
```

Comment:
why not fix the version? it is not a very reliable lib, as ti changed a few times already